externals/kyuubi-spark-sql-engine/src/test/scala/org/apache/spark/kyuubi/SparkSQLEngineDeregisterSuite.scala

```diff
@@ -75,7 +75,12 @@ class SparkSQLEngineDeregisterMsgSuite extends SparkSQLEngineDeregisterSuite {
       // Upper case SQL types in error messages
       // see https://issues.apache.org/jira/browse/SPARK-39007
       // Use double quotes for SQL configs in error messages
-      "to int causes overflow,to \"INT\" causes overflow")
+      // see https://issues.apache.org/jira/browse/SPARK-39214
+      // Improve errors related to CAST
+      """to int causes overflow,
+        |to "INT" causes overflow,
```

Comment:
We don't need this error message `to "INT" causes overflow` anymore, the behavior of uppercasing SQL types in error messages is something that happened after Spark-3.3.

Reply:
I simplify to just check "overflow"